blocklib/picoscope/test/qa_Picoscope.cc

```diff
@@ -12,140 +13,211 @@ using namespace std::string_literals;
 namespace fair::picoscope::test {
 
 // Replace with your connected Picoscope device
-template<typename T, AcquisitionMode aMode>
-using PicoscopeT = Picoscope4000a<T, aMode>;
+template<typename T>
+using PicoscopeT = Picoscope3000a<T>;
```

Comment:
Note: I think editing the test source to switch between different models is not very nice, the test should just be parameterized by the Picoscope type or we should have separate tests by model (that could potentially share the implementation, without having to recompile everything when working on a specific implementation). The tests should then just check whether the scope is connected and if not just skip the test.

The way it is now also means that the CI and regular rebuilds only builds the version currently committed, so we might break one of the other versions without noticing.

Since we still have to implement the ps6000, I think this can be changed then just as well and should not block this PR.